src/Options/docs.js

```diff
@@ -28,6 +28,7 @@
  * @property {Boolean} enableAnonymousUsers Enable (or disable) anonymous users, defaults to true
  * @property {Boolean} enableExpressErrorHandler Enables the default express error handler for all errors
  * @property {String} encryptionKey Key for encrypting your files
+ * @property {Boolean} enforcePrivateUsers Is true if Parse Server should set public read and write access on new Parse.Users to false
```

Comment:
@mtrezza are you happy with this description and the key name `enforcePrivateUsers`?

Reply:
Ideally we should use a similar wording in the security check as is in the options. How about:

> Is true if new users should be created without public read and write access.

Reply:
I knew you'd be able to come up with something better than I had

Reply:
Well, language style is somewhat subjective, so I try to only recommend a change on the basis for a potential benefit for the developer.

Reply:
I will keep that in mind in the future in regards to consistency. We definitely have room for improvement.

100%, but in my subjective opinion, your suggestions are objectively great 😉. I honestly couldn't think of how to name / describe this feature but didn't think about it too hard because I was expecting your suggestion for improvement anyway 😂